src/main/java/org/jacp/controller/CompetitionController.java

```diff
@@ -77,6 +77,16 @@ public ResponseEntity<CompetitionDto> startCompetition(@PathVariable Long compet
         return ResponseEntity.ok(competitionDto);
     }
 
+    @PostMapping("/{competitionId}/join")
+    public ResponseEntity<CompetitionDto> joinParticipants(@PathVariable Long competitionId,
```

Comment:
`joinParticipant` or just `join` for the method name will be more clear.

Reply:
Fixed